packages/core/src/languages/solidity/types.ts

```diff
@@ -318,13 +317,9 @@ export interface SolcInput {
 
 export interface CompilerOutputContract {
   abi: Array<any>
-  storageLayout?: SolidityStorageLayout
   evm: {
     bytecode: CompilerOutputBytecode
     deployedBytecode: CompilerOutputBytecode
-    methodIdentifiers: {
-      [methodSignature: string]: string
-    }
```

Comment:
I trimmed out everything that did not appear to be necessary from the `BuildInfo` type and then just made sure all the tests still passed. I still need to do live network testing with this, so I'm not totally sure these changes are all good, but it seems fine. Let me know if anything seems off though.